src/app/loadout/mod-assignment-drawer/selectors.ts

```diff
@@ -21,6 +21,7 @@ export function useEquippedLoadoutArmor(loadout: Loadout) {
       const allItems = allItemsSelector(state);
       const loadoutDimItems = [];
 
+      // TODO: if there's not an item in one of the slots, pick the current equipped!
```

Comment:
I meant "show the current equipped item in that bucket" so it's clearer what'll actually happen.